app/stylesheets/dashboard/charts-container.scss

```diff
@@ -1,5 +1,16 @@
 @import '../config/colours';
+@import '../config/dimensions';
+@import '../config/type';
 
 .charts-container {
   background: $grey-light;
 }
+
+.tab-label {
```

Comment:
my mistake, sorry! don't know why I didn't find that.

Ok great. Move `.tab-label` inside `charts-container {` then and we're good.